setup.cfg

```diff
@@ -20,11 +20,12 @@ classifiers =
 python_requires = >=3.6
 packages = find:
 include_package_data = True
+setup_requires = setuptools_scm
 install_requires =
-  numpy>=1.11
+  numpy>=1.13
   scipy
-  matplotlib>=1.3
```

Comment:
Even though they are really old does it make sense to keep these here, just in case someone installs sunpy in to an ancient env?

Reply:
Well for pandas, the numpy version probably limits some of them

I can add back the matplotib one